### PR TITLE
Update the AWS Upgrade information

### PR DESCRIPTION
**Pull request by ianf77**
Two stages of procedures added so far

### CODE REVIEW

File: stories/topics/proc-aap-aws-upgrade-pull-container-image.adoc [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+= Pull the ansible-on-clouds-ops 2.3 container image
+
+.Procedure
+. Pull the Docker image for Ansible on Clouds operational container with the same tag as the version you want to upgrade to.
++
+[NOTE]
+====
+The Ansible on Clouds operational image tag must match the version that you want to upgrade to. For example, if your foundation deployment version is 2.2.20230215, pull the operational image with tag 2.3.20230221-00 to upgrade to version 2.3.20230221-00.
+====
```
Comment (zkayyali812) [inline stories/topics/proc-aap-aws-upgrade-pull-container-image.adoc:11]: We should also add the AMEA region image pulling instruction too.

**Review by zkayyali812 — CHANGES_REQUESTED** [review]
2 minor comments

File: stories/topics/proc-aap-aws-backup-platform-stack.adoc [hunk 2]
```diff
@@ -0,0 +1,26 @@
+[id="proc-aap-aws-backup-platform-stack"]
+
+= Backing up your {PlatformNameShort} stack
+
+.Procedure
+. Pull the `ansible-on-clouds-ops 2.3` container image tag must match with theversion you want to upgrade to.
```
Comment (zkayyali812) [inline stories/topics/proc-aap-aws-backup-platform-stack.adoc:6]:
```suggestion
. Pull the `ansible-on-clouds-ops 2.2` container image tag must match with theversion you want to upgrade to.
```

**Review by zkayyali812 — APPROVED** [review]

**Review by ariordan-redhat — COMMENTED** [review]
Review not fully completed

File: stories/topics/ref-aws-update-upgrade-data-file.adoc [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+= Updating the data file
+
+You must populate the data file before triggering the upgrade. 
+The following variables are parameters listed in the data file.
+
+* `ansible_config_path` (Optional) Only use if overriding with a custom `ansible_config`.
+* `cloud_credentials_path` is the path to your AWS credentials file.
+* `deployment_name` is the name of the foundation deployment. 
+This is the same name you used when you deployed the foundation.
+* `aws_region`` is the region where the foundation deployment is located.
```
Comment (ariordan-redhat) [inline stories/topics/ref-aws-update-upgrade-data-file.adoc:12]:
```suggestion
* `aws_region` is the region where the foundation deployment is located.
```

File: stories/topics/proc-aap-aws-upgrade-pull-container-image.adoc [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
++
+[source,bash]
+----
+$ export IMAGE=_<image_path>_
+$ docker pull $IMAGE
+----
+
+for EMEA regions (Europe, Middle East, Africa) run the following command instead:
++
```
Comment (ariordan-redhat) [inline stories/topics/proc-aap-aws-upgrade-pull-container-image.adoc:20]:
```suggestion

```

The `+` shows up in the built doc.

File: stories/topics/proc-aap-aws-upgrade-pull-container-image.adoc [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+The Ansible on Clouds operational image tag must match the version that you want to upgrade to. For example, if your foundation deployment version is 2.2.20230215, pull the operational image with tag 2.3.20230221-00 to upgrade to version 2.3.20230221-00.
+====
++
+[source,bash]
+----
+$ export IMAGE=_<image_path>_
+$ docker pull $IMAGE
+----
+
+for EMEA regions (Europe, Middle East, Africa) run the following command instead:
```
Comment (ariordan-redhat) [inline stories/topics/proc-aap-aws-upgrade-pull-container-image.adoc:19]:
```suggestion
For EMEA regions (Europe, Middle East, Africa) run the following command instead:
```

File: stories/topics/proc-aws-from-stack-prepare-environment.adoc [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+====
++
+* `aws_s3_bucket` is the name of an S3 bucket that the playbook can access to upload a CloudFormation Template. The  name must not contain upper case letters.
+* `aws_efs_physical_id` is the physical Id of the EFS from the original deployment. 
+For example, `fs-06837574544929090`.
+* `aws_cf_keypair_name` is the keypair to pass as a parameter when creating the new restored deployment.
++
+[NOTE]
+====
+The keypair used must exist in the aws region you are restoring to.
```
Comment (ariordan-redhat) [inline stories/topics/proc-aws-from-stack-prepare-environment.adoc:61]:
```suggestion
The keypair used must exist in the AWS region you are restoring to.
```

File: stories/topics/proc-aws-generate-upgrade-data-file.adoc [hunk 7]
```diff
@@ -0,0 +1,36 @@
+[id="proc-aws-generate-upgrade-data-file_{concept}"]
+
+= Generating the upgrade data file
+
+The commands in this section create a directory and populate it with an empty data template that, when completed, is used during the upgrade. 
+
+Run the following commands to generate the required data file. 
+
+.Procedure
+. Create a folder to hold the configuration files
```
Comment (ariordan-redhat) [inline stories/topics/proc-aws-generate-upgrade-data-file.adoc:10]:
```suggestion
. Create a folder to hold the configuration files.
```

File: stories/topics/proc-aws-generate-upgrade-data-file.adoc [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
++
+[source,bash]
+----
+$ mkdir command_generator_data
+----
+. Populate the `command_generator_data` folder with configuration file template.
```
Comment (ariordan-redhat) [inline stories/topics/proc-aws-generate-upgrade-data-file.adoc:16]:
```suggestion
. Populate the `command_generator_data` folder with the configuration file template.
```

File: stories/topics/proc-aws-from-stack-pull-container-image.adoc [hunk 9]
```diff
@@ -0,0 +1,20 @@
...
+.Procedure
+. Pull the `ansible-on-clouds-ops` 2.2 container image with the same tag as the foundation deployment.
++
+[source,bash]
+----
+$ export IMAGE=quay.io/ansible-on-clouds/ansible-on-clouds-ops:2.2.20230215-00
+$ docker pull $IMAGE
+----
++
+for EMEA regions (Europe, Middle East, Africa) run the following command instead:
```
Comment (ariordan-redhat) [inline stories/topics/proc-aws-from-stack-pull-container-image.adoc:14]:
```suggestion
For EMEA regions (Europe, Middle East, Africa) run the following command instead:
```

**Review by ariordan-redhat — CHANGES_REQUESTED** [review]
Essential changes: concept -> context

The rest are just suggestions.

**Review by ariordan-redhat — CHANGES_REQUESTED** [review]
Essential changes: concept -> context

The rest are just suggestions.

File: stories/topics/proc-aap-aws-upgrade-pull-container-image.adoc [hunk 10]
```diff
@@ -0,0 +1,25 @@
...
+====
+The Ansible on Clouds operational image tag must match the version that you want to upgrade to. For example, if your foundation deployment version is 2.2.20230215, pull the operational image with tag 2.3.20230221-00 to upgrade to version 2.3.20230221-00.
+====
++
+[source,bash]
+----
+$ export IMAGE=_<image_path>_
+$ docker pull $IMAGE
+----
+
```
Comment (ariordan-redhat) [inline stories/topics/proc-aap-aws-upgrade-pull-container-image.adoc:18]:
```suggestion
+
```

File: stories/topics/proc-aap-aws-upgrade-pull-container-image.adoc [hunk 11]
```diff
@@ -0,0 +1,25 @@
+[id="proc-aap-aws-upgrade-pull-container-image"]
+
+= Pulling the ansible-on-clouds-ops 2.3 container image
+
+.Procedure
+. Pull the Docker image for Ansible on Clouds operational container with the same tag as the version you want to upgrade to.
```
Comment (ariordan-redhat) [inline stories/topics/proc-aap-aws-upgrade-pull-container-image.adoc:6]: Bullet point instead of number for a one-step procedure?

**Pull request merged**